app/notification.go

```diff
@@ -516,6 +525,17 @@ func GetExplicitMentions(post *model.Post, keywords map[string][]string) *Explic
 					}
 				}
 			}
+
+			// If word contains a multibyte character, check if it contains a multibyte keyword
+			if len(word) != utf8.RuneCountInString(word) {
+				for _, key := range multibyteKeywords {
+					if strings.Contains(word, key) {
```

Comment:
Since this is just checking if the text contains a keyword, will this run into any issues with word boundaries? For example, could you run into a case where a keyword like “car” matches “carrot”?

Reply:
It never happen if the keyword does not contain multibyte characters, because `key` in here must include multibyte characters. And original issue said that they want to find keyword `番茄` in `我爱吃番茄炒饭`, and it's achieved by this PR.
Note: If the keyword includes Alphabet and multibyte character, the keyword is considered as a multibyte keyword.

Reply:
Ah, I see. That makes sense then